tests/conftest.py

```diff
@@ -2,7 +2,11 @@
 import pytest_asyncio
 import tempfile
 from pathlib import Path
+from typing import Union, Any
 
+from _pytest.runner import CallInfo
+from _pytest.runner import CollectReport
+from _pytest.runner import TestReport
```

Comment:
It looks like these may all be available in the public API instead of `_pytest`.  https://github.com/pytest-dev/pytest/issues/7469

Perhaps not applicable after checking above, but I expect isort might prefer these to be all together?  If not we may want to adjust a setting for that.

Reply:
I didn't pay attention whether `pre-commit` ran `isort` for me or not. But it should have, right?

Reply:
```pycon
Python 3.9.5 (default, Jun  3 2021, 15:18:23) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytest
>>> pytest.__version__
'7.0.1'
>>> from pytest import CallInfo, CollectReport, TestReport
>>>
```

If you have latest pytest and it isn't working for you I would be curious to know.  We could dig into that on keybase.

```suggestion
from pytest import CallInfo, CollectReport, TestReport
```

---

On the pre-commit, it's default is to only run on 'modified' files.  I'm not sure the definition of that.  I personally just manually run it with `--all`.